### DISCUSSION TITLE
add sidebar links to another site?

### DISCUSSION THREAD

**Discussion by marksie1988** [cotes2020/jekyll-theme-chirpy#1826] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

Hi all, 

Is it possible to add a sidebar "tab" that links to another website? 

Basically i want to link out to my resume website from my blog

### Operations you have already tried

none

**Comment by kungfux** [comment]
Sidebar expects all tab links to be relative to the website, so it's not possible without customizing the logic.

However, I can suggest a workaround to use javascript to redirect to any page you want. Here is an example.

Create a file `tabs/resume.md` with the following content:

```markdown
---
layout: page
icon: fas fa-link
order: 6
---

<script>
window.location.href = "https://google.com";
</script>

```

**Reply by SoumyaK4** [reply]
@kungfux @marksie1988 I used this in ```_includes/sidebar.html```
```
<!-- coffee -->
        <li class="nav-item{% if tab.url == page.url %}{{ " active" }}{% endif %}">
          <a href="{{'https://buymeacoffee.com/SoumyaK4'}}" target="_blank" class="nav-link">
            <i class="fas fa-coffee"></i>
            <span>BUY ME A COFFEE</span>
          </a>
        </li>
```
[Source](https://github.com/SoumyaK4/WeiChi/blob/f3466dbe6471ad18ca4f750c651572050ed97038/_includes/sidebar.html#L41)

I was searching for how to do it in the forum then I remembered I did it a few months ago, stupid me XD

**Reply by kungfux** [reply]
That's true, it is possible to override the logic by replacing the chirpy file. However, the option you suggest comes with a maintenance cost. Every time you upgrade the blog to a newer Gem version, you may run into unexpected problems and have to redo your customizations. Sure, it's up to each of us to decide what works best.

**Reply by SoumyaK4** [reply]
@kungfux I see, I was wondering why my site suddenly doesn't work after upgrading gem.

even though both repo's have the same code now, I copied all files over to the new repo that I made using 7.1.1 it works lol, the one mentioned in source
